site/mkdocs.yml

```diff
@@ -48,6 +48,7 @@ nav:
     - How to Release: how-to-release.md
   - User docs:
     - Getting Started: getting-started.md
+    - Flink: flink.md
```

Comment:
Can we move documentation to a separate PR? There is no need to mix docs with build.

Reply:
Fine,  will propose another separate PR for the flink document.